projects/packages/jetpack-mu-wpcom/src/class-jetpack-mu-wpcom.php

```diff
@@ -53,4 +54,11 @@ public static function load_coming_soon() {
 			require_once __DIR__ . '/features/coming-soon/coming-soon.php';
 		}
 	}
+
+	/**
+	 * Load WP REST API plugins for wpcom
+	 */
+	public static function load_wpcom_rest_api_endpoints() {
+		wpcom_rest_api_v2_load_plugin_files( 'features/wpcom-endpoints/*.php' );
```

Comment:
Also, should we also do fields?
```suggestion
		wpcom_rest_api_v2_load_plugin_files( 'features/wpcom-endpoints/*.php' );
		wpcom_rest_api_v2_load_plugin_files( 'features/wpcom-fields/*.php' );
```
Or YAGNI, let someone add that when they do find a need for it?

Reply:
Yeah, let someone add it if/when they need to